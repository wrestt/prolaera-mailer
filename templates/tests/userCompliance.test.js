import fs from 'fs';
import renderer from 'react-test-renderer';
import userCompliance from '../builders/userCompliance';
import completeCompliance from './json/completeCompliance';

describe('Email compliance Report', () => {
  const logoUrl = 'https://assets.prolaera.com/prolaeraLogo_fullText.png';
  it('it returns an emails html', async () => {
    const emailHtml = await userCompliance(completeCompliance, logoUrl);
    expect(emailHtml).toBeDefined();
  });

  it('it checks snapshot with no set logo', async () => {
    const emailHtml = await userCompliance(completeCompliance, logoUrl);
    const emailComponent = renderer.create(emailHtml);
    const userComplianceJson = emailComponent.toJSON();
    expect(userComplianceJson).toMatchSnapshot();
  });

  it('it writes an html file', async () => {
    const email = await userCompliance(completeCompliance, logoUrl);
    const saved = await writeFile(email);
    expect(saved).toEqual(true);
  });
});

async function writeFile(emailHtml) {
  return new Promise((resolve, reject) => {
    fs.writeFile(`${__dirname}/test.html`, emailHtml, err => {
      if (err) {
        console.log(err);
        reject(err);
      }
      resolve(true);
    });
  });
}
