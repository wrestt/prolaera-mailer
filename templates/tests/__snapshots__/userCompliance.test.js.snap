// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Email compliance Report it checks snapshot with no set logo 1`] = `
"<!DOCTYPE html PUBLIC \\"-//W3C//DTD XHTML 1.0 Strict//EN\\" \\"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd\\"><html lang=\\"en\\" xmlns=\\"http://www.w3.org/1999/xhtml\\"><head><meta http-equiv=\\"Content-Type\\" content=\\"text/html; charset=utf-8\\"/><meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0\\"/><title>Compliance Report</title><style type=\\"text/css\\">@media
only screen and (max-width: 600px),
(min-device-width: 600px) and (max-device-width: 1024px)  {

	/* Force table to not be like tables anymore */
	table, thead, tbody, th, td, tr {
		display: block;
	}

	.emailHeader td {
		width: 100%;
	}

	.emailHeader td img {
		margin-left: auto;
		margin-right: auto;
	}

	.subHeader {
		display: relative;
	}

	.report td {
		/* Behave  like a &quot;row&quot; */
		border: none;
		border-bottom: 1px solid #eee;
    position: relative;
    text-align: center;
	}

	.report td:before {
		/* Now like a table header */
		position: absolute;
		/* Top/left values mimic padding */
		top: 6px;
		left: 6px;
		width: 45%;
		padding-right: 10px;
		white-space: nowrap;
	}
	
	.customButton:hover {
		color:black;
		
	}</style></head><body style=\\"width:100%;margin:0;padding:0;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%\\"><table width=\\"100%\\" height=\\"100%\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" align=\\"left\\" valign=\\"top\\"><tbody><tr><td align=\\"center\\" valign=\\"top\\"><table width=\\"600\\" align=\\"center\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><table class=\\"emailHeader\\" align=\\"center\\" width=\\"100%\\" style=\\"background-color:#F7F7F7\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><tr><td align=\\"center\\"><table align=\\"center\\" cellSpacing=\\"0\\" style=\\"padding:15px;max-width:584px\\"><tr align=\\"left\\"><td width=\\"292px\\"><img alt=\\"logo\\" src=\\"https://assets.prolaera.com/prolaeraLogo_fullText.png\\" width=\\"199.00305623471883\\" height=\\"47\\" style=\\"display:block;outline:none;border:none;text-decoration:none\\"/></td><td width=\\"252px\\" style=\\"text-align:center\\"><a href=\\"https://app.prolaera.com\\"> compliance report â†’ </a></td></tr></table></td></tr></tbody></table><table width=\\"100%\\" height=\\"100%\\" style=\\"background-color:#F7F7F7\\" cellPadding=\\"0\\" cellSpacing=\\"0\\" border=\\"0\\" align=\\"left\\" valign=\\"top\\"><tbody><tr><td><table cellSpacing=\\"20\\" align=\\"center\\" cellPadding=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><tr><td align=\\"center\\" style=\\"background-color:#FFFFFF\\"><div style=\\"padding:15px\\"><div><div style=\\"font-size:28px;text-align:center;line-height:25px\\"> CGFM</div><div style=\\"text-align:center;line-height:25px\\"><strong>Reporting Period: </strong> March 29, 2018 - March 28, 2020</div><div style=\\"text-align:center;padding-bottom:10px\\"><strong>License Expiration Date: </strong> March 28, 2020</div></div><table class=\\"report\\"><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>General Hours</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 7.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 80 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 72.5 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Year</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 7.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 40 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 32.5 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span><strong>Total</strong></span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 7.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 80 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#c0392b;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 72.5 </div></td></tr></table></div></td></tr><tr><td align=\\"center\\" style=\\"background-color:#FFFFFF\\"><div style=\\"padding:15px\\"><div><div style=\\"font-size:28px;text-align:center;line-height:25px\\"> AICPA</div><div style=\\"text-align:center;line-height:25px\\"><strong>Reporting Period: </strong> March 30, 2018 - March 29, 2021</div><div style=\\"text-align:center;padding-bottom:10px\\"><strong>License Expiration Date: </strong> March 29, 2021</div></div><table class=\\"report\\"><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>General Hours</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 7.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 40 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 32.5 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Year</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 7.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 40 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 32.5 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span><strong>Total</strong></span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 7.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 120 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#eea303;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 112.5 </div></td></tr></table></div></td></tr><tr><td align=\\"center\\" style=\\"background-color:#FFFFFF\\"><div style=\\"padding:15px\\"><div><div style=\\"font-size:28px;text-align:center;line-height:25px\\"> AICPA - CPA/ABV</div><div style=\\"text-align:center;line-height:25px\\"><strong>Reporting Period: </strong> January 1, 2018 - December 31, 2018</div><div style=\\"text-align:center;padding-bottom:10px\\"><strong>License Expiration Date: </strong> December 31, 2018</div></div><table class=\\"report\\"><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Business Valuation</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#f1c40f;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#f1c40f;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 20 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#f1c40f;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 15 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span><strong>Total</strong></span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#f1c40f;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#f1c40f;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 20 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#f1c40f;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 15 </div></td></tr></table></div></td></tr><tr><td align=\\"center\\" style=\\"background-color:#FFFFFF\\"><div style=\\"padding:15px\\"><div><div style=\\"font-size:28px;text-align:center;line-height:25px\\"> AICPA - EBPAQC</div><div style=\\"text-align:center;line-height:25px\\"><strong>Reporting Period: </strong> March 30, 2018 - March 29, 2021</div><div style=\\"text-align:center;padding-bottom:10px\\"><strong>License Expiration Date: </strong> March 29, 2021</div></div><table class=\\"report\\"><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Employee Benefits</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#64bb5d;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 0 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#64bb5d;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 8 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#64bb5d;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 8 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span><strong>Total</strong></span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#64bb5d;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 0 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#64bb5d;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 8 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#64bb5d;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 8 </div></td></tr></table></div></td></tr><tr><td align=\\"center\\" style=\\"background-color:#FFFFFF\\"><div style=\\"padding:15px\\"><div><div style=\\"font-size:28px;text-align:center;line-height:25px\\"> California CPA</div><div style=\\"text-align:center;line-height:25px\\"><strong>Reporting Period: </strong> February 10, 2017 - December 31, 2018</div><div style=\\"text-align:center;padding-bottom:10px\\"><strong>License Expiration Date: </strong> December 31, 2018</div></div><table class=\\"report\\"><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Technical</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 1162 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 36 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 0 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Non-Tech</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 7.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> N/A </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 0 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>General Ethics</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 0 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 4 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 4 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Regulatory Review</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 0 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> N/A </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 0 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Year</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 38.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 20 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 0 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span><strong>Total</strong></span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 1169.5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 80 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#16a085;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 0 </div></td></tr></table></div></td></tr><tr><td align=\\"center\\" style=\\"background-color:#FFFFFF\\"><div style=\\"padding:15px\\"><div><div style=\\"font-size:28px;text-align:center;line-height:25px\\"> Enrolled Agent</div><div style=\\"text-align:center;line-height:25px\\"><strong>Reporting Period: </strong> January 1, 2018 - December 31, 2020</div><div style=\\"text-align:center;padding-bottom:10px\\"><strong>License Expiration Date: </strong> December 31, 2018</div></div><table class=\\"report\\"><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Federal Taxes</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 66 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 61 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>General Ethics</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 0 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 6 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 6 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span><strong>Total</strong></span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 72 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#0e83cd;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 67 </div></td></tr></table></div></td></tr><tr><td align=\\"center\\" style=\\"background-color:#FFFFFF\\"><div style=\\"padding:15px\\"><div><div style=\\"font-size:28px;text-align:center;line-height:25px\\"> Prolaera CPA</div><div style=\\"text-align:center;line-height:25px\\"><strong>Reporting Period: </strong> April 2, 2018 - April 1, 2019</div><div style=\\"text-align:center;padding-bottom:10px\\"><strong>License Expiration Date: </strong> April 1, 2019</div></div><table class=\\"report\\"><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>General Hours</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 40 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 35 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span>Year</span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 40 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 35 </div></td></tr><tr><td colSpan=\\"4\\" style=\\"border-bottom:1px #333333 solid;padding-top:4px\\"></td></tr><tr><td colSpan=\\"4\\" style=\\"padding-bottom:3px\\"></td></tr></div><tr><td style=\\"font-size:16px;font-family:verdana,geneva,sans-serif;vertical-align:top\\"><span><strong>Total</strong></span></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Earned </div><div style=\\"font-size:36px\\"> 5 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Hours Required </div><div style=\\"font-size:36px\\"> 40 </div></td><td style=\\"font-family:arial;color:#ffffff;background-color:#702fa8;padding:26px 14px;border-left:5px solid #FFFFFF;text-align:center\\"><div style=\\"font-size:14px\\"> Remaining </div><div style=\\"font-size:36px\\"> 35 </div></td></tr></table></div></td></tr></tbody></table></td></tr></tbody></table><table align=\\"center\\" width=\\"100%\\" cellSpacing=\\"20\\" style=\\"background-color:#333333\\" cellPadding=\\"0\\" border=\\"0\\" valign=\\"top\\"><tbody><tr><td align=\\"center\\" style=\\"font-size:19px;font-family:Helvetica;color:#ffffff\\">Something not right? </td></tr><tr><td align=\\"center\\" style=\\"font-size:12px;font-family:Helvetica;color:#ffffff\\">You can contact us at <a href=\\"https://www.prolaera.com/contact/\\" target=\\"_blank\\" style=\\"color:#ffffff;text-decoration:underline;font-size:12px;font-family:Helvetica\\">Prolaera.com</a>
or email us directly at <a href=\\"mailto:Support@Prolaera.com\\" target=\\"_blank\\" style=\\"color:#ffffff;text-decoration:underline;font-size:12px;font-family:Helvetica\\">Support@Prolaera.com.</a></td></tr><tr><td align=\\"center\\" style=\\"font-size:12px;font-family:Helvetica;color:#ffffff\\">Powered By</td></tr><tr><td align=\\"center\\"><img alt=\\"react\\" src=\\"https://gallery.mailchimp.com/eb930e30a15249a18d658c183/images/196fa069-3113-464c-a304-a1aa9f99fe22.png\\" width=\\"100\\" height=\\"23\\" style=\\"display:block;outline:none;border:none;text-decoration:none\\"/></td></tr><tr><td align=\\"center\\" style=\\"font-size:12px;font-family:Helvetica;color:#ffffff\\">Â© Prolaera 2017</td></tr></tbody></table></tbody></table></td></tr></tbody></table></body></html>"
`;
